h2o-algos/src/test/java/hex/tree/PrintMojoTreeTest.java

```diff
@@ -185,11 +186,30 @@ private void assertMojoJSONEqualsFixture(Model model, String fixtureFile) throws
     assertFalse(treeJson.isEmpty());
     final String expectedTreeJson = IOUtils.toString(getClass().getResourceAsStream(fixtureFile));
     assertEquals(
-        removeH2OVersion(expectedTreeJson), 
-        removeH2OVersion(treeJson)
+            removeH2OVersion(expectedTreeJson),
+            removeH2OVersion(treeJson)
     );
   }
   
+  private void assertMojoPngGenerated(Model model, String outputFileName) throws IOException {
+    final File modelFile = folder.newFile();
+    model.exportMojo(modelFile.getAbsolutePath(), true);
+    final File treeOutput = folder.newFile(outputFileName);
+    try {
+      PrintMojo.main(new String[]{"--input", modelFile.getAbsolutePath(), "--output", treeOutput.getAbsolutePath(), "--format", "png"});
+      fail("Expected PrintMojo to call System.exit()");
+    } catch (PreventedExitException e) {
+      // expected
+    }
+    Path treeOutputPath = treeOutput.toPath();
+    Assert.assertTrue(Files.exists(treeOutputPath));
+    Files.walk(treeOutputPath)
+                .sorted(Comparator.reverseOrder())
+                .map(Path::toFile)
+                .forEach(File::delete);
+    Assert.assertFalse(Files.deleteIfExists(treeOutputPath));
```

Comment:
its nicer to use static imports for asserts

Reply:
will fix

Reply:
fixed